tests/test_ble_lifecycle_receive_targets.py

```
@@ -33,82 +33,96 @@ def _make_iface(monkeypatch: pytest.MonkeyPatch) -> Any:
     return _build_interface(monkeypatch, DummyClient(), start_receive_thread=False)
 
 
+def _reset_state_manager(iface: Any) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `iface`

(ANN401)